### PR TITLE
[13.0][ADD] connector_routific: New module for connect with Routific Platform

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT28665

With this module we allow to connect Odoo with Routific for manage the route optimization.

please @carlosdauden @sergio-teruel review this

### CODE REVIEW

**Review by chienandalu — CHANGES_REQUESTED** [review]
Tests aren't working in travis

File: connector_routific/security/res_groups.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!-- Copyright 2021 Tecnativa - Carlos Roca
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).-->
+<odoo>
+    <record id="group_routific_manager" model="res.groups">
```
Comment (pedrobaeza) [inline connector_routific/security/res_groups.xml]: Why creating a specific group for this and not binding it to an existing one? And now you will need `sudo` in all the operations that needs this data.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: connector_routific/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Copyright 2021 Tecnativa - Carlos Roca
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+{
+    "name": "Routific Connector",
+    "summary": "Connector for Routific Platform",
+    "version": "13.0.1.0.0",
+    "development_status": "Alpha",
```
Comment (chienandalu) [inline connector_routific/__manifest__.py]: This should change to stable as it's working on customer for a couple of months already, isn't it?

**Review by chienandalu — APPROVED** [review]
Minor comment

**Comment by CarlosRoca13** [comment]
Changed @chienandalu

**Review by sergio-teruel — APPROVED** [review]

**Comment by sergio-teruel** [comment]
Can we merge this PR?

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**